.circleci/config.yml

```diff
@@ -99,6 +99,47 @@ commands:
             python ./test_runner.py --suite=push -o $ENZOTEST_DIR --answer-name=$ANSWER_NAME --local --strict=high --verbose << parameters.flags >>
           fi
 
+  compile-test:
+    description: "Test that enzo compiles in a few different configurations."
+    steps:
+      - run: |
+          source $BASH_ENV
+          source $HOME/venv/bin/activate
+          ./configure
+          cd src/enzo
+          make machine-linux-gnu
+          make load-config-test-suite
+          # test various baryon/particle precisions
+          for prec in 32 64
+          do for part in 32 64 128
+          do
+          if [ $prec == $part ]; then
```

Comment:
I think we should still test the precision-32 and particles-32 case. But we shouldn't test the 64/64 case because that's the default.